react/features/filmstrip/components/web/Filmstrip.js

```diff
@@ -345,6 +345,7 @@ class Filmstrip extends PureComponent <Props> {
                     initialScrollTop = { 0 }
                     itemKey = { this._gridItemKey }
                     onItemsRendered = { this._onGridItemsRendered }
+                    overscanRowCount = { 1 }
```

Comment:
From the package documentation: 

>  By default, grid overscans by one item.

So do we really need to specify it? Or do you want to be explicit for extra readability?

Reply:
There are a few aspects here:

- readability as you said
- it turns out that even if you specify `0` it gets ignored and it set to `1` (this is a little bit unrelated)
- I saw some issues in the repo that the default is actually `2` and the documentation states `1`. This is fixed now but I thought it is better to explicitly specify it to be on the safe site!

Reply:
Makes sense!